### PR TITLE
Add reverb 🎤

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

**Comment by Vaibhavs10** [comment]
> Isn't https://huggingface.co/Revai/reverb-diarization-v2 run using pyannnote?

Yes but it's used in a non-standard way - https://github.com/revdotcom/reverb/blob/main/diarization/infer_pyannote3.0.py

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -516,6 +516,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"model.safetensors"`,
 	},
+	reverb: {
+		prettyLabel: "Reverb",
+		repoName: "Reverb",
+		repoUrl: "https://github.com/revdotcom/reverb",
+		filter: false,
+		countDownloads: `path_extension:"yaml" OR path_extension:"pt" OR path_extension:"bin"`,
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:524]: Looks like counting only `config.yaml` would be enough, isn't it? If we consider that loading the model requires to download the config. This way no need to have both .pt and .bin depending on the file format. Also, counting both yaml and weights files will lead to duplicate counts.

Also, good news, `config.yaml` is already counted by default when no rules are set (see [here](https://huggingface.co/docs/hub/en/models-download-stats#which-are-the-query-files-for-different-libraries)). That's why there are already downloads in https://huggingface.co/models?other=reverb

Comment (Wauplin) [reply packages/tasks/src/model-libraries.ts:524]:
```suggestion
```

TL;DR: this line can be removed

Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:524]: You're indeed correct. I hallucinated there being no `config.yaml` in speaker diarization repos.

**Review by Wauplin — APPROVED** [review]
Thanks!
